ospatch/yum_update.go

```diff
@@ -108,7 +108,11 @@ func RunYumUpdate(ctx context.Context, opts ...YumUpdateOption) error {
 
 	var pkgNames []string
 	for _, pkg := range fPkgs {
-		pkgNames = append(pkgNames, pkg.Name)
+		nameWithArch := pkg.Name
```

Comment:
it might be a good idea to create separate function to construct name, rather than do it in place, function like would encapsulate the logic effectively 
```
func packageNameFrom(name string, arch Architecture) {
...
}
```
or 

```
func packageNameFrom(pkg *PkgInfo) {
...
}
```